workflows/workflow_2.5d_preprocessing.nf

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env nextflow
+nextflow.enable.dsl=2
+
+/* S-OCT 2.5D Preprocessing */
+/* This workflow will generate the mosaic grids and compute the XY shifts between slices
+
+To execute this script, you can use the following command (example):
+
+nextflow run workflow_2.5d_preprocessing.nf --directory <PATH_TO_DATADIR> --output_directory . -resume
+
+Notes
+-----
+Copy this workflow in the processing directory.
+Add the -resume flag to resume the pipeline from the last successfully completed process.
+*/
+
+// Parameters
+params.directory = ""
+params.tiles_directory = params.directory + "/tiles"
+params.output_directory = params.directory
+params.mosaicgrids_directory = params.output_directory + "/mosaic_grids_2d"
+
+/* Processes */
+// Compute the shift between the slices
+process compute_xy_shifts{
+    input:
+        path directory
+    output:
+        path "shifts_xy.csv"
+    publishDir path: "${params.output_directory}", mode: "copy"
+    script:
+        """
+        linum_estimate_xyShift_fromMetadata $directory shifts_xy.csv
```

Comment:
these script names won't work with our new build system. You need the .py at the end.